# Request close gripper signal

---
bool success
